crates/bevy_pbr/src/light.rs

```diff
@@ -719,7 +721,8 @@ impl ClusterConfig {
             ClusterConfig::FixedZ {
                 total, z_slices, ..
             } => {
-                let aspect_ratio = screen_size.x as f32 / screen_size.y as f32;
+                let aspect_ratio: f32 =
+                    AspectRatio::new(screen_size.x as f32, screen_size.y as f32).into();
```

Comment:
IMO we should have a `from_pixels` method or something that takes the integer values.

Reply:
You mean in the `AspectRatio`?

Reply:
Yeah, an impl on this would be useful.

Reply:
Thought so, it's added in the latest commit.